ra/ra.go

```diff
@@ -361,6 +364,28 @@ func (ra *RegistrationAuthorityImpl) NewAuthorization(ctx context.Context, reque
 		}
 	}
 
+	if ra.reuseValidAuthz {
+		auths, err := ra.SA.GetValidAuthorizations(ctx, regID, []string{identifier.Value}, ra.clk.Now())
+		if err != nil {
+			outErr := core.InternalServerError("unable to get existing validations")
+			ra.log.Warning(string(outErr))
+		}
+
+		if existingAuthz, ok := auths[identifier.Value]; ok {
```

Comment:
We should also be checking that the authz has more than, say, a day left before it expires. I'm okay with hardcoding a day rather than adding another config param.